doc/manpages/mapping.asciidoc

```diff
@@ -0,0 +1,52 @@
+KAKOUNE(1)
+==========
+
+NAME
+----
+mapping - a
+
+Description
+-----------
+Creating and removing shortcuts boils down to the following functions,
+respectively:
+
+---------------------------------------
+map [flags] <scope> <mode> <key> <keys>
+unmap <scope> <mode> <key> [<expected>]
+---------------------------------------
+
+The *map* command makes *key* behave as if the *keys* sequence was typed.
+
+*mode* dictates in what context the mapping will be available:
+
+	*insert*::
+		insert mode (default: 'i')
```

Comment:
not sure what that `default: 'i'` means, I'd think insert mode is explicit enough